### DISCUSSION TITLE
kafkaUser

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [strimzi/strimzi-kafka-operator#6921] [Q&A]
if I want to give same user access to two topic do I just repeat the resource like this below?

apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaUser
metadata:
  name: test-user 
  labels:
    strimzi.io/cluster: test-cluster
spec:
  authentication:
    type: tls-external
  authorization:
    type: simple
    acls:
      - resource:
          type: topic
          name:  test1
          patternType: literal
        operation: All
        host: "*"
      - resource:
          type: topic
          name:  test2
          patternType: literal
        operation: All
        host: "*"

**Comment by scholzj** [comment]
You just add multiple rules to the `acls` list within the single `KafkaUSer` resource.
